hyprexpo/main.cpp

```diff
@@ -87,15 +87,25 @@ static void swipeUpdate(void* self, SCallbackInfo& info, std::any param) {
 
     info.cancelled = true;
     if (!swipeActive) {
-        if (g_pOverview)
-            return;
-        if ((**PPOSITIVE ? 1.0 : -1.0) * e.delta.y <= 0)
+        if (g_pOverview && (**PPOSITIVE ? 1.0 : -1.0) * e.delta.y <= 0) {
+            renderingOverview = true;
+            g_pOverview       = std::make_unique<COverview>(g_pCompositor->m_pLastMonitor->activeWorkspace, true);
+            renderingOverview = false;
+            gestured          = 300;
```

Comment:
@vaxerski I used 300 for the value here, but I don't know if that is correct

Reply:
any large value will do I believe, its just to snap iirc